allennlp/training/learning_rate_schedulers/combined.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, Any, List, Tuple, Optional
+
+from overrides import overrides
+import torch
+
+from allennlp.common.lazy import Lazy
+from allennlp.training.learning_rate_schedulers.learning_rate_scheduler import LearningRateScheduler
+
+
+@LearningRateScheduler.register("combined")
+class CombinedLearningRateScheduler(LearningRateScheduler):
+    """
+    This `LearningRateScheduler` can be used to apply an arbitrary number of other schedulers
+    one after the other.
+
+    These schedulers are defined though the `schedulers` parameter, which takes a list
+    of `Tuple[int, Lazy[LearningRateScheduler]]`. The first field of the tuple, the `int`,
+    specifies how many epochs the corresponding scheduler will be used before the next
+    scheduler takes its place.
```

Comment:
It would be neat if we could schedule these in a finer-grained way than per epoch, but it'll do. In fact, since we can just define epochs to be arbitrary numbers of steps, this might be all we ever need.